presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSymlinkInputFormat.java

```diff
@@ -87,12 +74,44 @@ public void testSymlinkTable()
                 "STORED AS " +
                 "INPUTFORMAT 'org.apache.hadoop.hive.ql.io.SymlinkTextInputFormat' " +
                 "OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'");
-
+        hdfsClient.createDirectory(warehouseDirectory + "/TestAvroSymlinkInputFormat/data");
+        saveResourceOnHdfs("avro/original_data.avro", warehouseDirectory + "/TestAvroSymlinkInputFormat/data/original_data.avro");
```

Comment:
put some extra file too in that directory (so that we can be sure it's content get skipped)